sre-recipes/sandboxctl

```diff
@@ -99,7 +102,26 @@ def main(action, recipe_name):
     elif action == 'hint':
         logging.info('Giving hint for %s', recipe_name)
         recipe.hint()
+    elif action == 'loadgen':
+        logging.info('Restarting Load Generator in Step Pattern')
+        restart_loadgen(pattern=LoadPattern.STEP)
```

Comment:
2 things:

1) Are we starting loadgen automatically when the cluster completed provisioning?
2) Do we only support step pattern now? If so, we should at least add what are the next steps for the user in the logging message: go to the URL X to change hatching rate, etc.

Reply:
Good catch, this is actually left over from Gloria's implementation. I split the `loadgen` into it's own command branch (below), rather than grouping it in with the sre recipes. I'll remove this block

As for your questions:
1) the loadgenerator will start as normal with the basic pattern
2) the user can switch between patterns with `sandboxctl loadgen basic` and `sandboxctl loadgen step`. There isn't too much documentation around this yet, so maybe we should add a print statement after making a change or something?